synapse/api/auth.py

```diff
@@ -638,22 +635,24 @@ def _get_user_from_macaroon(self, macaroon_str):
                 errcode=Codes.UNKNOWN_TOKEN
             )
 
-    def validate_macaroon(self, macaroon, type_string, additional_validation_functions):
+    def validate_macaroon(self, macaroon, type_string, verify_expiry):
```

Comment:
A docstring here to explain why `verify_expiry` would ever be `false` would be good.